src/lib/buildContentRelationshipField.ts

```diff
@@ -4,7 +4,6 @@ type BuildContentRelationshipFieldConfig<
 	Document extends prismic.PrismicDocument,
 > = {
 	document: Document;
-	text?: string;
```

Comment:
Related question from `@prismicio/client`: https://github.com/prismicio/prismic-client/pull/352#discussion_r1758100722

Reply:
@angeloashmore can we merge this one into the main one?

Reply:
Yes, I'll merge and give another review with everything together.